src/parser/text_lexer.ml

```diff
@@ -105,7 +105,7 @@ let id_char =
     | 'a' .. 'z'
     | 'A' .. 'Z'
     | '!' | '#' | '$' | '%' | '&' | '\'' | '*' | '+' | '-' | '.' | '/' | ':'
-    | '<' | '=' | '>' | '?' | '@' | '\\' | '^' | '_' | '`' | '|' | '~' | '*' )]
+    | '<' | '=' | '>' | '?' | '@' | '\\' | '^' | '_' | '`' | '|' | '~' )]
```

Comment:
Why are you removing this case? Are the tests still passing while removing it? I believe it is allowed to write `(func $f*f (param i32) ...` and now it is not anymore?

Reply:
Oh this is because it was duplicated, right?

Reply:
It seems that we already have an `'*'` in the pattern (line 107)

Reply:
> Oh this is because it was duplicated, right?

Yes :) ocaml-lsp-server seems unable to detect it probably because it's in the extension node `[%sedlex.regexp?]`? When I copied this pattern to an OCaml function ocaml-lsp-server complained